### PR TITLE
lightning: add doc for send-kv-size

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

As tittle

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/14176
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -155,8 +155,14 @@ addr = "172.16.31.10:8287"
 #  - remove: records all duplicate records in the target table to the lightning_task_info database, like the 'record' algorithm.
 #    But it removes all duplicate records from the target table to ensure a consistent state in the target TiDB.
 # duplicate-resolution = 'none'
-# The number of KV pairs sent in one request in the physical import mode.
-# send-kv-pairs = 32768
+# The maximum number of KVs in one request when sending data to TiKV in physical import mode.
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:158]:
```suggestion
# The maximum number of KV pairs in one request when sending data to TiKV in physical import mode.
```

**Review by D3Hunter — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -155,8 +155,14 @@ addr = "172.16.31.10:8287"
 #  - remove: records all duplicate records in the target table to the lightning_task_info database, like the 'record' algorithm.
 #    But it removes all duplicate records from the target table to ensure a consistent state in the target TiDB.
 # duplicate-resolution = 'none'
-# The number of KV pairs sent in one request in the physical import mode.
-# send-kv-pairs = 32768
+# The maximum number of KV pairs in one request when sending data to TiKV in physical import mode.
+# Starting from v7.2.0, this parameter is deprecated and no longer takes effect after it is set.
+# If you want to adjust the amount of data sent to TiKV in one request, use the `send-kv-size` parameter instead.
+# send-kv-pairs = 3200
```
Comment (D3Hunter) [inline tidb-lightning/tidb-lightning-configuration.md:161]:
```suggestion
# send-kv-pairs = 32768
```

Comment (Frank945946) [reply tidb-lightning/tidb-lightning-configuration.md:161]: 这个上次成选说让改成 3200 了

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -155,8 +155,14 @@ addr = "172.16.31.10:8287"
...
+# The maximum size of one request when sending data to TiKV in physical import mode.
```
Comment (Frank945946) [inline tidb-lightning/tidb-lightning-configuration.md:162]: @D3Hunter MB 是不是也支持，建议给出几个写法

File: tidb-lightning/tidb-lightning-configuration.md (hunk 2 above)
Comment (hfxsd) [reply tidb-lightning/tidb-lightning-configuration.md:161]: 已经在这个 PR 里改成 3200 了  https://github.com/pingcap/docs-cn/pull/13993/files#diff-e60eb511e8e7fcd107a7d3a7d3da20c1e422686d723c4f14446d0c05cb22c08b

File: tidb-lightning/tidb-lightning-configuration.md (hunk 3 above)
Comment (D3Hunter) [reply tidb-lightning/tidb-lightning-configuration.md:162]: 之后统一补吧，store-write-bwlimit 也没有

File: tidb-lightning/tidb-lightning-configuration.md (hunk 2 above)
Comment (hfxsd) [reply tidb-lightning/tidb-lightning-configuration.md:161]: 经过讨论，先保留 send-kv-pairs = 32768

File: tidb-lightning/tidb-lightning-configuration.md (hunk 3 above)
Comment (Frank945946) [reply tidb-lightning/tidb-lightning-configuration.md:162]: ok

**Review by Frank945946 — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
/lgtm

**Pull request merged**